pkg/constants/constants.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2021 The Rode Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package constants
```

Comment:
I was tired of seeing the same constants being defined in multiple packages, so I started extracting some of them out.  This isn't an exhaustive list of all constants, and I only refactored the constants I touched during this PR.  Ill do a follow up later to extract the rest of them.